qiskit/visualization/circuit_visualization.py

```diff
@@ -454,7 +465,13 @@ def _latex_circuit_drawer(
                     if filename.endswith(".pdf"):
                         os.rename(base + ".pdf", filename)
                     else:
-                        image.save(filename, "PNG")
+                        try:
+                            image.save(filename)
+                        except ValueError as ve:
+                            raise ValueError(
```

Comment:
I think this error should be changed as well based on @1ucian0's comment :)

Reply:
Made the changes.